src/views/docs-view/index.tsx

```diff
@@ -7,19 +7,27 @@ import dynamic from 'next/dynamic'
 import classNames from 'classnames'
 import { useCurrentProduct } from 'contexts'
 import DocsViewLayout from 'layouts/docs-view-layout'
-import defaultMdxComponents from 'layouts/sidebar-sidecar/utils/_local_platform-docs-mdx'
 import DevDotContent from 'components/dev-dot-content'
 import DocsVersionSwitcher from 'components/docs-version-switcher'
-import { DocsViewProps, ProductsToPrimitivesMap } from './types'
+import { DocsViewProps } from './types'
 import { NoIndexTagIfVersioned } from './components/no-index-tag-if-versioned'
 import ProductDocsSearch from './components/product-docs-search'
+import getDocsMdxComponents from './utils/get-docs-mdx-components'
 import s from './docs-view.module.css'
 
 /**
  * Layouts
  *
- * note: layout in frontmatter is not supported yet, this is early stage work.
+ * Note: layout in frontmatter is not supported yet, this is early stage work.
  * Asana task: https://app.asana.com/0/1202097197789424/1202850056121889/f
+ *
+ * Note: this layout logic is used for non-`/docs` landing pages.
+ * The `/docs` landing pages use `ProductRootDocsPathLanding`.
+ * We could consider this approach, as it implies arbitrary layouts
+ * are supported via frontmatter. We could also consider renaming the
+ * "layout" property to something like "contentLayout", as otherwise
+ * it seems like it could be confused with broader "src/layouts" code.
+ * Task: https://app.asana.com/0/1202097197789424/1204069295311480/f
```

Comment:
I got confused here, despite the fact that IIRC i think I implemented some of this stuff (😅), so added a task here to try to make the flow here a little more clear in the future.